packages/manager/modules/pci/src/projects/project/sidebar/sidebar.scss

```diff
@@ -0,0 +1,4 @@
+.oui-status.oui-badge_new {
+  background-color: #47fffa !important;
```

Comment:
Prefer using a more specific selector instead of `!important`

Reply:
fixed